pkg/scheduler/flavorassigner/tas_flavorassigner.go

```diff
@@ -57,13 +56,11 @@ func (a *Assignment) WorkloadsTopologyRequests(wl *workload.Info, cq *schdcache.
 }
 
 func (psa *PodSetAssignment) HasFailedNode(wl *workload.Info) bool {
```

Comment:
```suggestion
func (psa *PodSetAssignment) HasUnhealthyNode(wl *workload.Info) bool {
```

Could we rename `FaileNode` to `UnhealthyNode` at all?